data/txHandler_test.go

```diff
@@ -953,6 +959,7 @@ func TestTxHandlerProcessIncomingCacheRotation(t *testing.T) {
 // TestTxHandlerProcessIncomingCacheBacklogDrop checks if dropped messages are also removed from caches
 func TestTxHandlerProcessIncomingCacheBacklogDrop(t *testing.T) {
 	partitiontest.PartitionTest(t)
+	t.Parallel()
```

Comment:
This test checks the global variable `transactionMessagesDroppedFromBacklog`
One day, it may fail when another test changes its value. 
And this is a supper duper fast less than 10ms.